src/frontend/src/handler/variable.rs

```diff
@@ -41,17 +41,11 @@ pub fn handle_set(
 
 pub(super) fn handle_show(context: OptimizerContext, variable: Vec<Ident>) -> Result<RwPgResponse> {
     let config_reader = context.session_ctx.config();
-    if variable.len() != 1 {
-        return Err(
-            ErrorCode::InvalidInputSyntax("only one variable or ALL required".to_string()).into(),
-        );
-    }
-    // TODO: Verify that the name used in `show` command is indeed always case-insensitive.
-    let name = &variable[0].value.to_lowercase();
+    let name = variable.iter().map(|e| e.real_value()).join("_");
```

Comment:
now we support config with multiple words.

Reply:
In PostgreSQL's document, I don't find that `show` command can receive multiple arguments, although `show transaction isolation level;` or `show time zone;` can actually run successfully. In order to follow PostgreSQL's syntax, I recommend checking whether variable is "TRANSACTION ISOLATION LEVEL" here instead of using `join("_")`. `join("_")` will allow some input such as "show search path;" or "show query mode;", which are not allowed in PostgreSQL.

Reply:
Good advice

Reply:
Please check again @HuaHuaY . Thanks!